mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java

```diff
@@ -67,6 +65,7 @@ protected void onCreate(Bundle savedInstanceState) {
             //enabling passCodeListener only when user has already setup PassCode
             passCodeView.setPassCodeListener(this);
         }
+
```

Comment:
Remove all the changes in this file except the `finish()` statement.

Reply:
done!